docs/topics/installing-vs-code-extension.adoc

```diff
@@ -0,0 +1,38 @@
+// Module included in the following assemblies:
+//
+// * docs/vsc-extension-guide/master.adoc
+
+
+[id="installing-vs-code-extension_{context}"]
+= Installing the {ProductShortName} extension for Visual Studio Code
+
+You can install the {ProductShortName} extension for Visual Studio Code (VS Code) on any platform that supports VS Code.
+
+.Prerequisites
+
+* 8 GB RAM
+* JDK 11 or later
+
+
+.Procedure
+
+. Set the environmental variable `JAVA_HOME` to `jdk11`:
++
+[source, terminal]
+----
+$ export JAVA_HOME=jdk11
+----
+
+. In VS Code, click the *Extensions* icon on the Activity bar to open the *Extensions* view.
```

Comment:
I'm not sure that Extensions view or Extensions icon are marked up. They are bold if there is a label or a tooltip. I can't recall an Extensions view label. Does it say "Extensions" somewhere when you are in the Extensions view? Do you understand what I am saying?

Reply:
If I'm not mistaken, you told me to bold these names 10 days ago (I've unhidden your previous comment). Did I misunderstand?

Reply:
In that case, I must have checked. Otherwise I would not have told you to bold them. So you can ignore this comment. I will resolve it.